src/Arrays/MPIStateArrays.jl

```diff
@@ -368,12 +368,12 @@ function begin_ghost_exchange!(Q::MPIStateArray; dependencies = nothing)
     __Irecv!(Q)
 
     # wait on (prior) MPI sends
-    @tic mpi_sendwait
+    # @tic mpi_sendwait
```

Comment:
Easier to just define empty macros in TicToc than to comment all these? Anyway, will hopefully find a solution today.